x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/resilient.ts

```diff
@@ -301,19 +295,20 @@ export default function resilientTest({ getService }: FtrProviderContext) {
             })
             .then((resp: any) => {
               expect(resp.body).to.eql({
-                connector_id: simulatedActionId,
+                connector_id: resilientActionId,
                 status: 'error',
-                retry: false,
+                retry: true,
+                message: 'an error occurred while running the action',
                 errorSource: TaskErrorSource.FRAMEWORK,
-                message:
-                  'error validating action params: types that failed validation:\n- [0.subAction]: expected value to equal [getFields]\n- [1.subAction]: expected value to equal [getIncident]\n- [2.subAction]: expected value to equal [handshake]\n- [3.subActionParams.incident.name]: expected value of type [string] but got [undefined]\n- [4.subAction]: expected value to equal [incidentTypes]\n- [5.subAction]: expected value to equal [severity]',
+                service_message:
+                  '[Action][IBM Resilient]: Unable to create incident. Error: Incident name is required.',
```

Comment:
Same here.

Reply:
Because createIncident is an abstract method of CaseConnector. It doesn't have a separate schema schema.